### ISSUE TITLE
Dragging of non-line and inline editable blocks do not save their position

### ISSUE THREAD

**Issue by clarkepaul** [dnadesign/silverstripe-elemental#548]
When you have a mix of in-page editable blocks and blocks like the Form Block which aren't in-page editable, and you re-order them the blocks do not save their position.

Tested in 4.3 (pre stable release)

cc @brynwhyman

**Comment by clarkepaul** [comment]
Could be problems with block pages in blocks pages if the above test okay?

**Comment by ScopeyNZ** [comment]
Spoke about this with @sachajudd . I think this is mostly just #547 (no preview update is triggered on re-order)? I've raised an PR that resolves that.

I've also included a fix to update the draft status of blocks that you re-order.

There is another issue that we've been experiencing which appears to be fixed by recent core changes - an update of installed modules resolved that issue.

Once #553 is merged I'd say this is closed - reopen if you feel the issue is still not resolved.

**Comment by ScopeyNZ** [comment]
#553 is merged. I'll close this for now - feel free to reopen if there are some steps that can reproduce the issue as reported.

Thanks!

**Comment by asecondwill** [comment]
I'm getting similar issues to @lexakami  with ordering not working in SS4, Elemental 4.0

**Comment by kinglozzer** [comment]
Same here (elemental 4.0.0), I put it down to a one-off blip as I’d made a large number of model changes at the time. I’m back on that project later this week so I’ll take a good look then if no one beats me to it. For reference, none of the blocks in my project are inline-editable.

**Comment by ScopeyNZ** [comment]
Okay thanks to @Kethatril on the community Slack I've found the probable cause for the issue. We were updating surrounding objects when sorting things but only on the "draft" table and never the "live" table. I've raised a PR at #580 that should resolve this issue.

**Comment by robbieaverill** [comment]
#580 is merged
